js/adoptMain.js

```diff
@@ -0,0 +1,11 @@
+let cartTotal = 0;
+let dogCost = 0;
+
+function dogDescription(name, breed, cost) {
+  alert(`This is ${name}, a ${breed} breed.  $${cost} to adopt!`)
```

Comment:
To be more technically precise, you could add a ; to the end of your alert(); if you want.